packages/create-bison-app/template/_templates/graphql/new/graphql.ejs

```diff
@@ -3,20 +3,28 @@ to: graphql/modules/<%= name %>.ts
 ---
 <% camelized = h.inflection.camelize(name) -%>
 <% plural = h.inflection.pluralize(camelized) -%>
-import { objectType, extendType, inputObjectType, stringArg, arg, nonNull, enumType } from 'nexus';
-import { Role } from '@prisma/client';
-import { UserInputError, /*ForbiddenError*/ } from 'apollo-server-micro';
+import { objectType, extendType, inputObjectType, list, arg, nonNull /*, enumType*/ } from 'nexus';
 
-// import { isAdmin } from '../services/permissions';
+import { isAdmin } from '../../services/permissions';
+
+// Example ENUM Type
+// import { Role } from '@prisma/client';
+// export const UserRole = enumType({
+//   name: 'Role',
+//   members: Object.values(Role),
+// });
 
 // <%= camelized %> Type
 export const <%= camelized %> = objectType({
   name: '<%= camelized %>',
   description: 'A <%= camelized %>',
   definition(t) {
+    // Database Specific Fields
     t.nonNull.id('id');
     t.nonNull.date('createdAt');
     t.nonNull.date('updatedAt');
+    // Model Specific Fields
+    t.nonNull.string('name');
```

Comment:
I like the idea of trying to organize some of this. I worry that doing it this way may actually confuse things though. It requires you to ask yourself "is this a database field or a model field". Additionally, the `name` field that you have here doesn't have a custom resolver so it looks like it would be coming from the database.

Perhaps it would be better not to force the organization and just show an example of a custom resolver for a field?

Reply:
Yup, those comments are a bit misleading. 
`name` is in our generated example. 

I'll remove the comments and leave it as is.